src/Aquifer.Common/Middleware/ApiKeyAuthorizationMiddleware.cs

```diff
@@ -41,4 +47,15 @@ public async Task InvokeAsync(HttpContext context, ICachingApiKeyService caching
 public class ApiKeyAuthorizationMiddlewareOptions
 {
     public ApiKeyScope Scope { get; set; }
+
+    /// <summary>
+    ///     Routes that will not require an API key. This uses simple <c>StartsWith</c> logic.
```

Comment:
`Path.StartsWithSegments(x)` is [not quite that simple](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.pathstring.startswithsegments?view=aspnetcore-9.0#microsoft-aspnetcore-http-pathstring-startswithsegments(microsoft-aspnetcore-http-pathstring)) due to `/` handling.